pkg/models/shortUrl.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import (
+	"errors"
+)
+
+var (
+	ErrShortUrlNotFound = errors.New("Short URL not found")
+)
+
+type ShortUrl struct {
```

Comment:
```suggestion
type ShortURL struct {
```

Reply:
thanks so much for all the feedback! this was the only one i (intentionally) didn't rename, as it caused the table name to be resolved as `short_u_r_l`